### PR TITLE
Add veteran info chapter to burials form

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3130

screen shot 2017-06-08 at 11 46 05 am
screen shot 2017-06-08 at 11 46 10 am

screen shot 2017-06-08 at 11 46 19 am
screen shot 2017-06-08 at 11 46 23 am

### CODE REVIEW

File: src/js/burials/config/form.js [hunk 1]
```diff
@@ -67,6 +79,66 @@ const formConfig = {
           }
         }
       }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
```
Comment (jbalboni) [inline src/js/burials/config/form.js]: I could probably use the common applicantInformation page for this, but I'd have to add a couple more configurable bits and it seemed like it was stretching the purpose of that shared page too much.

File: src/js/burials/config/form.js [hunk 2]
```diff
@@ -67,6 +79,66 @@ const formConfig = {
...
+          title: 'Deceased Veteran information',
+          path: 'veteran-information',
+          uiSchema: {
+            'ui:title': 'Deceased Veteran information',
+            veteranFullName: fullNameUI,
+            'view:veteranId': personId.uiSchema(
+              'veteran',
+              'view:veteranId.view:noSSN',
+              'I don’t have the Veteran’s Social Security number'
```
Comment (cehsu) [inline src/js/burials/config/form.js]: Should "Number" in "Social Security Number" be capitalized throughout? You did not introduce this convention here, just wondering if we should follow the labeling instructions @stacyhlinn provided more closely.

**Review by cehsu — APPROVED** [review]
LGTM.

File: src/js/burials/config/form.js (hunk 2 above)
Comment (jbalboni) [inline src/js/burials/config/form.js]: I believe the guidance we've gotten before from @peggygannon and/or the previous copy person was that "number" should be lower case.

**Pull request merged**